bimmer_connected/vehicle/vehicle_status.py

```diff
@@ -90,6 +90,12 @@ def all_lids_closed(self) -> bool:
         # pylint:disable=missing-function-docstring
         return self.vehicle.doors_and_windows.all_lids_closed
 
+    @property
+    @deprecated("vehicle.doors_and_windows.roof_closed")
+    def roof_closed(self) -> bool:
+        # pylint:disable=missing-function-docstring
+        return self.vehicle.doors_and_windows.roof_closed
+
```

Comment:
Please remove this, this is only for backwards-compatibilty of old, existing properties.